src/headless-cms/components/utils.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  ArticleType,
+  Card,
+  Collection,
+  GeneralCollectionType,
+  getCollectionCards,
+  PageType,
+} from 'react-helsinki-headless-cms';
+
+import {
+  getCmsArticlePath,
+  getCmsPagePath,
+} from '../../domain/app/routes/utils';
+
+export function getCmsCollectionList(
+  collections: GeneralCollectionType[]
+): React.ReactElement<typeof Collection>[] {
+  return collections.map((collection) => (
+    <Collection
+      key={`collection-${Math.random()}`}
+      title={collection.title}
+      collectionContainerProps={{
+        withDots: collection.items.length < 4 ? false : true,
+      }}
+      type="grid"
+      cards={getCollectionCards(collection as GeneralCollectionType).map(
+        (item) => (
+          <Card
+            key={`card-${item.id}`}
+            {...item}
+            clampText={true}
+            withShadow={true}
+            hasLink={true}
+            url={getCollectionItemUrl(item)}
+            // imageLabel={item.featuredImage?.node?.title ?? ''}
+            // imageUrl={item.featuredImage?.node?.mediaItemUrl ?? ''}
```

Comment:
Left intentionally?

Reply:
I thought those might give some hint for next refactoring, so I left them there. Some refactoring and further HCRC-lib integration is definitely needed in future.